Aijeyomah/Book-club
# continuous integration setup

#### What does this PR do?
setup continuous in app 
 
#### Description of Task to be completed?
- Setup continuous integration with travis ci and test coverage analysis with coveralls.
- Write unit test.

#### How should this be manually tested?
- Clone the repo and cd into the project folder
- Checkout to ch-basic-config branch and run npm install.
- Run npm test to see results.
